addons/survey/models/survey_survey.py

```diff
@@ -45,7 +45,7 @@ def _get_default_session_code(self):
         return False  # could not generate a code
 
     # description
-    title = fields.Char('Survey Title', required=True, translate=True)
```

Comment:
I would not remove the string of this field, instead I would use "nolabel" in the view if you need to hide the label